### PR TITLE
Add version 5.22.0-rc.1

### PR DESCRIPTION
**Pull request by hush-hush**

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"5.22.0-rc.1": {
+		"AGENT_VERSION": "5.22.0-rc.1",
+		"AGENT_BRANCH": "5.22.0-rc.1",
+		"AGENT6_BRANCH": "6.0.0-beta.3",
```
Comment (olivielpeau) [inline release.json]: not sure this parameter is useful, but we should probably set it to the latest beta or `6.0.0-beta.7` like 5.21.1

File: release.json [hunk 2]
```diff
@@ -10,6 +10,17 @@
...
+		"INTEGRATIONS_CORE_BRANCH": "5.22.0-rc.1",
+		"TRACE_AGENT_BRANCH": "5.20.0",
+		"TRACE_AGENT_ADD_BUILD_VARS": "true",
+		"PROCESS_AGENT_BRANCH": "5.20.2",
```
Comment (olivielpeau) [inline release.json]: nit: shouldn't this be `5.21.0` for now?

File: release.json [hunk 3]
```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "datadog-5.5.0",
 		"OMNIBUS_SOFTWARE_BRANCH": "master"
 	},
+	"5.22.0-rc.1": {
+		"AGENT_VERSION": "5.22.0-rc.1",
+		"AGENT_BRANCH": "5.22.0-rc.1",
+		"AGENT6_BRANCH": "6.0.0-beta.3",
+		"INTEGRATIONS_CORE_BRANCH": "5.22.0-rc.1",
+		"TRACE_AGENT_BRANCH": "5.20.0",
```
Comment (olivielpeau) [inline release.json]: nit: `5.21.1` for now I guess

**Comment by gmmeyer** [comment]
🎉

**Pull request merged**